lib/cli_logging.dart

```diff
@@ -4,12 +4,11 @@
 
 /// This library contains functionality to help command-line utilities to easily
 /// create aesthetic output.
+library cli_logging;
```

Comment:
As it seems we're all steering away from library declarations it's a little jarring to see some added in but if that's the best way to support package-level docs, I can live with it.  (And honestly don't feel too strongly either way.)